bazel/README.md

```diff
@@ -84,6 +84,40 @@ Bazel can also be built with the Docker image used for CI, by installing Docker
 See also the [documentation](https://github.com/envoyproxy/envoy/tree/master/ci) for developer use of the
 CI Docker image.
 
+## Building Envoy with Remote Execution
+
+Envoy can also be built with Bazel [Remote Executioon](https://docs.bazel.build/versions/master/remote-execution.html),
+part of the CI is running with the hosted [GCP RBE](https://blog.bazel.build/2018/10/05/remote-build-execution.html) service.
+
+To build Envoy with a remote build services, run Bazel with your remote build service flags and with `--config=remote-clang`.
+For example the following command runs build with the GCP RBE service used in CI:
+
+```
+bazel build //source/exe:envoy-static --config=remote-clang \
+    --remote_cache=grpcs://remotebuildexecution.googleapis.com \
+    --remote_executor=grpcs://remotebuildexecution.googleapis.com \
+    --remote_instance_name=projects/envoy-ci/instances/default_instance
+```
+
+Change the value of `--remote_cache`, `--remote_executor` and `--remote_instance_name` for your remote build services. Tests can
+be run in remote execution too.
+
+Note: Currently the test run configuration in `.bazelrc` doesn't download test binaries and test logs,
```

Comment:
downloading failed test logs by default seems really useful, particularly if something flakes. If you can't get the log then you may be out of luck. 

However, downloading the test binaries seems less useful and more expensive. Is there a way to tease these options apart?

Reply:
That is what I'm asking in https://github.com/bazelbuild/bazel/issues/8934

Reply:
Cool. You might want to reference that bug in a comment here?

Reply:
actually that issue is closed now; can you update the default to download test logs?

Reply:
The fix is not in released bazel yet.